src/plugins/app-route/handler.js

```diff
@@ -30,7 +30,6 @@ export default (languageRenderers: { [string]: LanguageRenderer$ }) => (
 					.redirect(renderResult.redirect.url)
 					.permanent(Boolean(renderResult.redirect.permanent));
 			}
-			request.trackSession();
```

Comment:
no longer making a separate tracking call on first render - tracking is handled entirely by `trackApi`, which is called on first render and subsequent API proxy calls